packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -987,32 +987,32 @@ export function getElementPadding(element) {
 
 /**
  * Returns the absolute height of an element.
+ * Uses getBoundingClientRect() to ensure synchronous layout calculation.
  * @param {Element} element
  * @returns {number} The absolute height of the element including padding.
  */
 export function getAbsoluteHeight(element) {
   if (!element) {
     return 0;
   }
-  const paddings = getElementPadding(element);
-  const padding = paddings.top + paddings.bottom;
 
-  // @ts-ignore
-  return Math.ceil(element.offsetHeight + padding);
+  // Using getBoundingClientRect ensures accurate height measurements across all browsers, especially Safari.
+  const elementBoundingRect = element.getBoundingClientRect();
+  return Math.ceil(elementBoundingRect.height);
 }
 
 /**
  * Returns the absolute width of an element.
+ * Uses getBoundingClientRect() to ensure synchronous layout calculation.
  * @param {Element} element
  * @returns {number} The absolute width of the element including padding.
  */
 export function getAbsoluteWidth(element) {
```

Comment:
This one is used in the `QuanticTabBar` as well IIRC to see in the InsightPanel if we need to collapse some tabs under the "More" dropdown.

Make sure this still works, both with Locker and LWS if you want to change this.

If there are some automated tests for that exact scenario that could be the proof as well.